fiftyone/server/routes/camera.py

```
@@ -0,0 +1,166 @@
+
+from starlette.endpoints import HTTPEndpoint
+from starlette.exceptions import HTTPException
+from starlette.requests import Request
+
+import fiftyone.core.odm.utils as fou
+from fiftyone.server import utils
+
+logger = logging.getLogger(__name__)
+
+
+def _get_dataset(dataset_id: str):
```

Ruff (0.14.10)

Missing return type annotation for private function `_get_dataset`

(ANN202)